es6.md

```diff
@@ -145,3 +145,49 @@ interface ClassExpression <: Class, Expression {
     type: "ClassExpression";
 }
 ```
+
+# Modules
+
+## ImportDeclaration
+
+**FIXME:** Esprima's behavior differs from Acorn's and SpiderMonkey's behavior in that `source` is of type `ModuleSpecifier`, which is otherwise identical to a `Literal`. See [Esprima bug 1077](https://github.com/jquery/esprima/issues/1077).
+
+```js
+interface ImportDeclaration <: Declaration {
+    specifiers: [ ImportSpecifier ];
+    source: Literal; // Or ModuleSpecifier? (Esprima deviation)
```

Comment:
Spec references that part of the `ImportDeclaration` (and the `ExportDeclaration` as well) as a `ModuleSpecifier`, assuming it will evolve over time. Really, what I'm trying to convey here is the idea that after the `from`, we will alway have a module specifier that could be a literal, rather than saying after the `from` we will have a literal, or something else.